<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="settingFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="settingLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="aLabel">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Force Applied between every adjacent Node according to hook's law.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Resting Length</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="mechanicalForceSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="rLabel">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Force Applied between every non-adjacent Node according to coulomb law.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Electric charge</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="electricalForceSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1,1,1,1,2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="NodeSizeLabel">
             <property name="text">
              <string>Node Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="nodeSizeSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Edge Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="EdgeSizeSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Reduce velocity of each Node in each animation frame by specified factor.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>DRAG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="dragSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Maintain each Node velocity less than equal to specified.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Max Speed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="maxSpeedSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="animationLabel">
             <property name="text">
              <string>Animation-Speed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="animationSpeedSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFit_In_View"/>
    <addaction name="actionzoom_in"/>
    <addaction name="actionzoom_out"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuget_adjacency_mat_of_scene">
     <property name="title">
      <string>get adjacency matrix of scene</string>
     </property>
     <addaction name="actionWithout_Bracket"/>
     <addaction name="actionWith_Bracket"/>
    </widget>
    <widget class="QMenu" name="menuget_Incidence_matrix_of_scene">
     <property name="title">
      <string>get Incidence matrix of scene</string>
     </property>
     <addaction name="actionWithout_Bracket_2"/>
     <addaction name="actionWith_Bracket_2"/>
    </widget>
    <addaction name="actionExport_Scene"/>
    <addaction name="menuget_adjacency_mat_of_scene"/>
    <addaction name="menuget_Incidence_matrix_of_scene"/>
   </widget>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>New</string>
    </property>
    <addaction name="actionLoad_From_adjacency_Matrix"/>
    <addaction name="actionLoad_From_Incidence_Matrix"/>
    <addaction name="actionClear_Scene"/>
    <addaction name="actionDemo"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionNo_Editing"/>
    <addaction name="actionInsert_Node"/>
    <addaction name="actionRemove_Node"/>
    <addaction name="actionConnect_Node"/>
    <addaction name="actionDisconnect_Node"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuMouse_Shortcut">
     <property name="title">
      <string>Mouse Shortcut</string>
     </property>
     <addaction name="actionCtrl_Drag_to_move_into_Scne"/>
     <addaction name="actionCtrl_mousWheel_to_Zoom_In_Out"/>
    </widget>
    <addaction name="menuMouse_Shortcut"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionFit_In_View">
   <property name="text">
    <string>Fit-In-View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionExport_Scene">
   <property name="text">
    <string>Export as Image</string>
   </property>
   <property name="toolTip">
    <string>Export scene as a image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="toolTip">
    <string>Setting</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_From_adjacency_Matrix">
   <property name="text">
    <string>Load From adjacency Matrix</string>
   </property>
  </action>
  <action name="actionWithout_Bracket">
   <property name="text">
    <string>Without Bracket</string>
   </property>
  </action>
  <action name="actionWith_Bracket">
   <property name="text">
    <string>With Bracket</string>
   </property>
  </action>
  <action name="actionInsert_Node">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add Node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove_Node">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove Node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionConnect_Node">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect Node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDisconnect_Node">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disconnect Node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionNo_Editing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No Editing</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionClear_Scene">
   <property name="text">
    <string>Clear Scene</string>
   </property>
   <property name="toolTip">
    <string>Clear scene</string>
   </property>
  </action>
  <action name="actionLoad_From_Incidence_Matrix">
   <property name="text">
    <string>Load From Incidence Matrix</string>
   </property>
  </action>
  <action name="actionWithout_Bracket_2">
   <property name="text">
    <string>Without Bracket</string>
   </property>
  </action>
  <action name="actionWith_Bracket_2">
   <property name="text">
    <string>With Bracket</string>
   </property>
  </action>
  <action name="actionzoom_in">
   <property name="text">
    <string>zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionzoom_out">
   <property name="text">
    <string>zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCtrl_Drag_to_move_into_Scne">
   <property name="text">
    <string>Ctrl + Drag to move into Scene</string>
   </property>
  </action>
  <action name="actionCtrl_mousWheel_to_Zoom_In_Out">
   <property name="text">
    <string>Ctrl + mousWheel to Zoom-In-Out</string>
   </property>
  </action>
  <action name="actionDemo">
   <property name="text">
    <string>Demo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
